### PR TITLE
Improve existing content

### PR DESCRIPTION
**Pull request by adamsilver**
## Context

There's a number of little content and style tweaks that have fallen out of alignment with the latest designs. This PR looks to fix all of those.

## Changes to respond offer

Before:
image

After:
image

## Changes to conditions page

Before:
image

After:
image

## Changes to confirm offer page

Before:
image

After: 
image

## Changes to reject application page

Before:
image

After:
image

## Changes to confirm rejection page

Before:
image

After:
image

## Changes to edit response page

Before:
image

After:
image

## Changes to withdraw page

Before:
image

After:
image

## Changes to withdraw confirmation page

Before:
image

After:
image

## Changes to confirming conditions met

Before:
image

After:
image

## Changes to confirming conditions not met

Before:
image

After:
image

## Help needed

- I don't know how to stop the ‘change response’ labels from being bold - fixed thanks to @stevehook 
- I don't know how to change the ‘withdraw offer’ text box error message from ‘can't be blank’ - fixed thanks to @duncanjbrown 

## Link to Trello card

https://trello.com/c/ZoTJfDpi/1682-update-production-content-to-match-prototype-content

### CODE REVIEW

**Comment by tvararu** [comment]
Suggestion: if this ends up living a while as it accumulates *all the nits*, then it will start to merge conflict. Usually it's easier to create multiple smaller PRs and merge them quickly. That being said :shipit:

**Comment by duncanjbrown** [comment]
Agree with @tvararu - this is plenty to ship in one go. Let me know if you want a hand fixing any specs

**Comment by adamsilver** [comment]
@tvararu @duncanjbrown - yep I agree. I'll avoid this in future. But please don't make me split it out. 🙏

File: config/locales/en.yml [hunk 1]
```diff
@@ -135,10 +135,14 @@ en:
           attributes:
             conditions_met:
               blank: Please specify if the candidate has met the conditions of the offer
+        withdraw_offer:
+          attributes:
+            offer_withdrawal_reason:
+              blank: Explain why you're withdrawing the offer
```
Comment (tvararu) [inline config/locales/en.yml]:
```suggestion
              blank: Explain why you’re withdrawing the offer
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -135,10 +135,14 @@ en:
...
         reject_application:
           attributes:
             rejection_reason:
-              blank: Enter feedback for the candidate
+              blank: Explain why you're rejecting the application
```
Comment (tvararu) [inline config/locales/en.yml]:
```suggestion
              blank: Explain why you’re rejecting the application
```

Comment (adamsilver) [reply config/locales/en.yml]: Ha, I had already done that locally :)

File: config/locales/en.yml (hunk 1 above)
Comment (adamsilver) [reply config/locales/en.yml]: Ha, I'd already done that locally :)

**Pull request merged**